### DISCUSSION TITLE
quickfilter wrong date format in dataexplorer

### DISCUSSION THREAD

**Discussion by teafmartins** [serenity-is/Serenity#6146] [Help] [answered]
Hi,

I'm using correct globalization culture in web.config and everything works fine with date in quickfilter (date format dd/mm/yyyy).
But I have problems with datetime quickfilter in dataexplorer. It assumes different date format (mm/dd/yyyy). Why this happen only happens in dataexplorer quickfiter?

I appriciate any help. Thx

**Comment by VictorTomaili** [comment]
DataExplorer is a premium feature. Please use StartSharp for premium support. If you don't have a licence, please check [this link](https://serenity.is/#Pricing) and play fair.

**Comment by teafmartins** [comment, marked as answer]
Ok, yes I have. Done.
